### ISSUE TITLE
Using filename from contentDisposition header

### ISSUE THREAD

**Issue by RobertFalk** [mrin9/RapiDoc#501]
When downloading zip files our server returns the target file name in the content disposition header. Rapidoc grabs that name and uses it, but the filename is in quotes and those quotes aren't stripped from the filename before it is used which messes up the download.  The code is [here](https://github.com/mrin9/RapiDoc/blob/50518f69ff8eaac0686032aa3894f3402abe3ab2/src/components/api-request.js#L1273), and I think all that needs to be done is to strip the double quotes with something like:
` me.respContentDisposition = me.respContentDisposition.replace(/"/g, '');`

We also have a related issue in that the generated curl command does not support the content disposition header.  It is possible for curl commands to support this header by, for example, adding the -OJ options, but in rapiDoc there currently is no way to configure the curl command based on the current api endpoint.  Would it be possible to add some form of curl configuration support that can be varied for different endpoints?

**Comment by mrin9** [comment]
thanks for taking time to point to the issue. would you be able to confirm if the filename having `quotes` in `content-disposition` is stemming from the server that you are using.
This is what I found out upon a quick search https://stackoverflow.com/questions/43756538/why-does-the-filename-of-the-contentdisposition-add-extra-slashes-and-quotes-e
and the web-server you are suing on your end

**Comment by RobertFalk** [comment]
I went back and tested and yes, if I remove the quotes from the filename it works.  We are using an apache server but I don't think that matters here since we add the quotes in our own gateway code so I can easily remove them. That approach is fine for now, but I'd be nice if RapiDoc supported quotes - the link you sent seems to indicate that quoted filenames is the standards-compliant approach.  Just to be sure I also tried curl and that works both with and without quoted filenames.
